### ISSUE TITLE
there's data leak in your code

### ISSUE THREAD

**Issue by ericyue** [yuxiangalvin/DeepLOB-Model-Implementation-Project#1]
could you try to set shuffle=False when split the train/valid  dataset and update the accuracy?  
In my dataset, I found if disable shuffle, the performance will drop a lot .

**Comment by MickyDowns** [comment]
Eric may be right. Shuffling is the default behavior for both sklearn train_test_split and keras.fit. Latter shouldn't matter as it only shuffles train. Former, however, shuffles the entire data set before splitting train / test. While generally helpful in avoiding local minima, shuffling time series in a model designed to find temporal relationships shouldn't be helpful. Here it is suggesting leakage as accuracy falls if you set train_test_split shuffle = False. The effect is more pronounced here than in the original paper as your y-encoding is more noisy (and realistic).
